src/algorithm/branching.jl

```diff
@@ -180,9 +180,8 @@ function branching_score(
     parent_result::NodeResult,
     config::AlgorithmConfig{StrongBranching},
 )
-    sb_model = config.branching_rule.strong_branching_model
+    sb_model = _default_lp_solver_factory(state, config)
```

Comment:
You want to use `config.lp_solver_factory` instead, in case the user sets an odd parameter, a small time limit, etc.